apple-pie/index.html

```diff
@@ -1,39 +1,80 @@
 <!DOCTYPE html>
 <html lang="en">
+  
   <head>
-    <meta charset="UTF-8" />
+    
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Apple Pie Recipe</title>
     <!-- Remember to link your styles -->
+    <link rel="stylesheet" href="styles/style.css" />
+    
   </head>
   <body>
-    Apple Pie
+    <section class="header">
+      <img class="apple-pie" src="/Users/maraline/Desktop/lab-css-recipes-clone/apple-pie/images/apple-pie.jpg" alt="" />
```

Comment:
```suggestion
      <img class="apple-pie" src="images/apple-pie.jpg" alt="" />
```
Esto si no estoy en tu ordenador no me funcionaria jejeje, si le ponemos images/.... mejor :D

Reply:
vale, es que lo de las imágenes no me salia de ninguna manera y lo deje asì direcamente para que me digas como es.  =(